pkg/deploy/predeploy.go

```diff
@@ -26,9 +27,13 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/keyvault"
 )
 
-// Rotate the secret on every deploy of the RP if the most recent
-// secret is greater than 7 days old
-const rotateSecretAfter = time.Hour * 168
+const (
+	// Rotate the secret on every deploy of the RP if the most recent
+	// secret is greater than 7 days old
+	rotateSecretAfter    = time.Hour * 168
```

Comment:
nit: It would be easier to figure out the timespan using `time.Hour * 24 * 7`

Reply:
Thanks, done.